Trakt/Api/DataContracts/BaseModel/TraktEpisode.cs

```diff
@@ -5,13 +5,13 @@ namespace Trakt.Api.DataContracts.BaseModel
     [DataContract]
     public class TraktEpisode
     {
-        [DataMember(Name = "season")]
```

Comment:
They made a change after the plugin was updated. This properties accepted a null in the past.